test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -993,6 +993,22 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
             $response.Output.Encoding.EncodingName | Should Be $expectedEncoding.EncodingName
             $response.Output | Should BeOfType 'Microsoft.PowerShell.Commands.BasicHtmlWebResponseObject'
         }
+
+        It "Verifies Invoke-WebRequest defaults to UTF8 on application/json when no charset is present" {
+            # when contenttype is set, WebListener suppresses charset unless it is included in the query
+            $query = @{
+                contenttype = 'application/json'
+                body        = '{"Test": "Test"}'
```

Comment:
Should we add multibyte characters to ensure Utf8 encoding works?

Reply:
No need. We are not testing the ability of .NET to properly decode UTF-8. We are testing the ability of the web cmdlets to chose the correct default encoding for application/json. This is an acceptable tests because we already have tests for the `iso-8859-1` defaults for other mime type and we have UTF-8 tests which do ensure we can work with UTF-8 characters. Here we only need to make sure the UTF-8 encoding was selected.